guides/source/action_view_overview.md

```diff
@@ -456,14 +494,14 @@ By default, templates will accept any `locals` as keyword arguments. To define w
 <%= message %>
 ```
 
-Rendering the partial without a `:message` local variable argument will raise an exception:
+The above makes `message` a required local variable. Rendering the partial without a `:message` local variable argument will raise an exception:
```

Comment:
This part confuses me. It says above that `message` is required but below that it is missing, so will raise an error?

Reply:
The above is a partial that needs to be passed the `message` when it is rendered. Below we are trying to render the partial *without* passing in `message` hence the error. The point this is trying to make is that it is an error to render partials without passing in required locals (as declared in the partial file itself with that magic comment). Does that make sense? Let me know if not and I will try to see if there is a way to reword or rearrange something in the examples...